src/cedar.js

```diff
@@ -127,6 +127,9 @@ var Cedar = function Cedar(options){
   //xhrs are in progress
   this._methodQueue=[];
 
+  // Set a base timeout
+  this._timeout = options.timeout || undefined;
```

Comment:
I would argue that this line is not needed, but to be consistent w/ other internals, I'd at least change it to `this._timeout = undefined;`